data/mods/Magiclysm/monsters/forgedwellers.json

```
@@ -94,9 +97,53 @@
     "vision_day": 30,
     "armor_fire": 50,
     "harvest": "demihuman_large_fur",
+    "death_function": { "effect": { "id": "summon_vengeance", "hit_self": true } },
     "special_attacks": [ { "type": "spell", "spell_data": { "id": "demon_breath_fire", "min_level": 7 }, "cooldown": 10 }, [ "scratch", 15 ] ],
     "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "BASHES", "ANIMAL", "PATH_AVOID_DANGER_1", "NO_BREATHE" ]
   },
+  {
+    "id": "mon_spirit_vengeance",
+    "type": "MONSTER",
+    "name": { "str": "vengeance spirit" },
+    "description": "It looks like a will o the wisp, darting back and forth, until it catches sight of its prey.  When the sickly green balefires light up inside the eyeholes of a skull, the mouth opens and a deep moan vibrates up through the legs of those standing nearby.",
```

```suggestion
    "description": "It looks like a will-o-the-wisp, darting back and forth, until it catches sight of its prey.  When the sickly green balefires light up inside the eyeholes of a skull, the mouth opens and a deep moan vibrates up through the legs of those standing nearby.",
```
So it looks like a will-o-the-wisp until it catches sight of prey, then it looks like a skull with sickly green fires inside the eyeholes?